docs/support_playbook.md

```diff
@@ -98,61 +98,57 @@ Make sure you know which part you are amending. Add `\r\n\r\n` for carriage retu
 ApplicationForm.find(_id).update!(becoming_a_teacher: 'new text', subject_knowledge: 'new_text', audit_comment: 'Updating grade following a support request, ticket ZENDESK-LINK')
 ```
 
-## Offers
-
-### Make or change offer
+## Courses and course locations 
 
-If the current application status is `awaiting_provider_decision` use MakeAnOffer service.
-
-If the current application status is `offer` use ChangeOffer service.
+### Changing a course or course location 
 
-### Change provider/course
+A provider may request that a candidate be placed on an different course, or a different site.
```

Comment:
could we indicate that this an application in `awaiting_provider_decision` and not already in the offered state?

Reply:
ah also I forgot about applications in the interviewing state, do we want to add something there?

Reply:
I've made the change @cwrw 

But given we now have a service in progress (https://github.com/DFE-Digital/apply-for-teacher-training/pull/6083) this guidance will become out of date shortly. @despo shall I close this PR?